app/src/common/shared/org/mozilla/vrbrowser/WidgetManagerDelegate.java

```diff
@@ -18,4 +18,5 @@
     void popBackHandler(Runnable aRunnable);
     void fadeOutWorld();
     void fadeInWorld();
+    void setTrayVisible(boolean visible);
```

Comment:
Tray already had a method to make it visible or invisible: Toggle(bool aEnabled)